<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditDialog</class>
 <widget class="QDialog" name="TextEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Text</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonFind">
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditFind">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButtonReplace">
           <property name="text">
            <string>Replace</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEditReplace">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="checkBoxCaseSensitive">
           <property name="text">
            <string>Case Sensitive</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>157</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pushButtonOK">
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pushButtonCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
